.github/workflows/testing.yml

```diff
@@ -79,7 +79,66 @@ jobs:
         run: |
           test/run-tests.sh
 
-  windows-testing:
+  macos-static:
+    strategy:
+      matrix:
+        # Our setup makes Clang segfault (YES.) on macOS 11.
+        # Let's not do that.
+        #os: [macos-11, macos-12]
+        os: macos-12
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install deps
+        shell: bash
+        run: |
+          ./.github/scripts/install_deps.sh ${{ matrix.os }}
+      # The `export` line is to allow working on macOS...
+      # Apple's base version is severely outdated, not even supporting -Wall,
+      # but it overrides Homebrew's version nonetheless...
```

Comment:
```suggestion
      # Export `bison` to allow using the version we install from Homebrew,
      # instead of the outdated one preinstalled on macOS (which doesn't
      # even support `-Wall`...).
```